### PR TITLE
add function to test if exact version of an app is installed

### PR DESCRIPTION
**Pull request by kpaul**
the changes for Get-CertsFromLocation and Get-CertThumbprint those changes were reviewed in a separate branch that has been merged with master.

### CODE REVIEW

File: common/Fabric-Install-Utilities.psm1 [hunk 1]
```diff
@@ -346,6 +369,21 @@ function Get-DecryptedString($encryptionCertificate, $encryptedString){
 	}
 }
 
+function Get-CertsFromLocation($certLocation){
```
Comment (michaelvidal24) [inline common/Fabric-Install-Utilities.psm1]: wasn't this function already merged?

File: common/Fabric-Install-Utilities.psm1 [hunk 2]
```diff
@@ -346,6 +369,21 @@ function Get-DecryptedString($encryptionCertificate, $encryptedString){
...
+    $currentLocation = Get-Location
+    Set-Location $certLocation
+    $certs = Get-ChildItem
+    Set-Location $currentLocation
+    return $certs
+}
+
+function Get-CertThumbprint($certs, $selectionNumber){
```
Comment (michaelvidal24) [inline common/Fabric-Install-Utilities.psm1]: same question as above...

File: common/Fabric-Install-Utilities.psm1 [hunk 3]
```diff
@@ -190,6 +190,29 @@ function Test-Prerequisite($appName, $minVersion)
...
+    if($installedAppResults -eq $null){
+        return $false;
+    }
+
+    if($supportedVersion -eq $null)
+    {
+        return $true;
+    }
+
+    $minVersionAsSystemVersion = [System.Version]$supportedVersion
```
Comment (michaelvidal24) [inline common/Fabric-Install-Utilities.psm1]: should we change this variable to represent more what it is, perhaps supportedVersionAsSystemVersion?

File: common/Fabric-Install-Utilities.psm1 (hunk 1 above)
Comment (kpaul) [reply common/Fabric-Install-Utilities.psm1]: yes, it was accidentally included here

File: common/Fabric-Install-Utilities.psm1 (hunk 2 above)
Comment (kpaul) [reply common/Fabric-Install-Utilities.psm1]: same answer as above

**Pull request merged**